.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: goreleaser
+
+on:
+  push:
+    # run only against stable semver tags on main
+    branches:
+      - main
+    tags:
+      - 'v[0-9]+\.[0-9]_\.[0-9]+'
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Fetch all tags
+        run: git fetch --force --tags
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Run GoReleaser
```

Comment:
Are we going to make use of goreleaser-pro?

Reply:
I don't think we'll need anything that's not provided by the free version.  Will circle back when/if it becomes necessary.